### PR TITLE
Add discovery to HEOS configuration options

### PR DESCRIPTION
**Pull request by andrewsayre**
**Description:**
Add discovery to HEOS configuration options

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#22652

**Pull request in [netdisco](https://github.com/home-assistant/netdisco):** home-assistant/netdisco#250

## Checklist:
- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by andrewsayre** [comment]
Should not be merged until _both_ parents are merged.

File: source/_components/heos.markdown [hunk 1]
```diff
@@ -24,7 +24,7 @@ The Heos component integrates [HEOS](http://heosbydenon.denon.com) capable produ
 
 ## {% linkable_title Configuration %}
 
-To add a Denon HEOS speaker to your installation, add the following to your `configuration.yaml` file:
+HEOS devices are discovered and setup automatically when the [discovery](/components/discovery) component is enabled. Alternatively, the component can be setup through the frontend control pannel integrations page or manually by adding the following to your `configuration.yaml` file:
```
Comment (Kane610) [inline source/_components/heos.markdown]:
```suggestion
HEOS devices are discovered and setup automatically when the [discovery](/components/discovery) component is enabled. Alternatively, the component can be setup through the frontend control panel integrations page or manually by adding the following to your `configuration.yaml` file:
```

File: source/_components/heos.markdown [hunk 2]
```diff
@@ -13,7 +13,7 @@ ha_release: 0.92
 ha_iot_class: Local Push
 ---
 
-The Heos component integrates [HEOS](http://heosbydenon.denon.com) capable products, such as speakers, amps, and receivers (Dennon and Marantz) into Home Assistant. Features currently include:
+The HEOS component integrates [HEOS](http://heosbydenon.denon.com) capable products, such as speakers, amps, and receivers (Dennon and Marantz) into Home Assistant. Features currently include:
```
Comment (Kane610) [inline source/_components/heos.markdown]:
```suggestion
The HEOS component integrates [HEOS](http://heosbydenon.denon.com) capable products, such as speakers, amps, and receivers (Denon and Marantz) into Home Assistant. Features currently include:
```

File: source/_components/heos.markdown (hunk 1 above)
Comment (andrewsayre) [reply source/_components/heos.markdown]: Fixed!  Thanks!

File: source/_components/heos.markdown (hunk 2 above)
Comment (andrewsayre) [reply source/_components/heos.markdown]: Fixed!  Thanks!

**Review by Kane610 — APPROVED** [review]

**Comment by andrewsayre** [comment]
Parents merged!

**Pull request merged**